docsets/mypy/docset.json

```diff
@@ -1,12 +1,25 @@
 {
     "name": "mypy",
-    "version": "0.9.1",
-    "archive": "mypy.tgz",
+    "version": "1.9.0",
+    "archive": "versions/mypy.1.9.tgz",
```

Comment:
You shouldn't use versions for this path. The main docset archive should be outside the versions folder.